src/devices/tuya.ts

```diff
@@ -4930,6 +4930,66 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE204_ves1ycwx'}],
+        model: 'SPM02',
+        vendor: 'TuYa',
+        description: 'Smart energy monitor for 3P+N system',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [e.voltage_phase_a(), e.voltage_phase_b(), e.voltage_phase_c(),
+            e.current_phase_a(), e.current_phase_b(), e.current_phase_c(),
+            e.power_phase_a(), e.power_phase_b(), e.power_phase_c(),
+            // Change the description according to the specifications of the device
+            e.energy().withDescription('Total forward active energy'),
+            e.produced_energy().withDescription('Total reverse active energy'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'energy', tuya.valueConverter.divideBy100],
+                [2, 'produced_energy', tuya.valueConverter.divideBy100],
+                [6, null, {
+                    from: (v: Buffer) => {
```

Comment:
Can you add a new valueConverter in tuya.ts for this and also use it for the `SPM01`? (to prevent code duplication)